plugin.py

```diff
@@ -1,10 +1,12 @@
 from hansken_extraction_plugin.api.extraction_plugin import ExtractionPlugin
 from hansken_extraction_plugin.api.plugin_info import Author, MaturityLevel, PluginId, PluginInfo
 from hansken_extraction_plugin.runtime.extraction_plugin_runner import run_with_hanskenpy
+from hansken_extraction_plugin.test_framework.test_plugin import _test_validate_standalone
```

Comment:
I think this import is unused?

Reply:
True, was also there for testing purposes.